ranger/README.md

```diff
@@ -0,0 +1,29 @@
+# Apache Ranger
+
+This initialization action installs [Apache Ranger](https://ranger.apache.org/) on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+ Apache Ranger enables monitoring and managing data security across the Hadoop ecosystem and uses [Apache Solr](http://lucene.apache.org/solr/) for audits.
+
+## Using this initialization action
+
+You can use this initialization action to create a new Dataproc cluster with Apache Ranger installed:
+
+1. Use the `gcloud` command to create a new cluster with this initialization action. 
+The following command will create a new high availability cluster named `<CLUSTER_NAME>`.
+
```

Comment:
Is HA required to run Ranger? If so, mention the reason; otherwise, use non-HA. Also, I noticed in the code, it doesn't enforce HA.

Reply:
HA configuration is not required however, it's recommended because Ranger is using Solr for auditing and in HA clusters Solr is deployed in SolrCloud mode which brings parallelization for querying data and for long-running clusters there could be a huge amount of data (ranger documentation is recommending to have at least 1TB free). 
I will change this example cluster creation code to a standard/single cluster and add proper info in notes paragraph.